### PR TITLE
Add search radius field to entities list

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: a553022adf6287af5c72839e1eb62e86e5f7eeb2 available at: https://gist.github.com/66ba63faaa75cc816135<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5665.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5665.dmg

**Comment by cozza13** [comment]
Make the default 100m. Also when I tested on 'sandbox' nothing showed up until I set it to 10000m, the phone booth on sandbox is within 20m.

**Comment by ctrlaltdavid** [comment]
Default increased to 100m.
Regarding things not showing up: remember that there's a bug that the entities list isn't refreshed when you first display it or subsequently tab to it - Worklist 20648.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0399c5ca355c56efc25cc5c88d784c605f0e1e00 available at: https://gist.github.com/66ba63faaa75cc816135<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5665.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5665.dmg

**Comment by cozza13** [comment]
OK. This is good to go. We can address the items not showing in Worklist 20648

File: examples/libraries/entityList.js [hunk 1]
```diff
@@ -33,7 +35,7 @@ EntityListTool = function(opts) {
 
     that.sendUpdate = function() {
         var entities = [];
-        var ids = Entities.findEntities(MyAvatar.position, 100);
+        var ids = Entities.findEntities(MyAvatar.position, searchDiameter);
```
Comment (AndrewMeadows) [inline examples/libraries/entityList.js]: **EntityScriptingInterface::findEntities()** takes a **radius** as the second argument, not a **diameter**:

    QVector<QUuid> EntityScriptingInterface::findEntities(const glm::vec3& center, float radius) const

**Comment by ctrlaltdavid** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: e2ee074b7a5890aa28e9ce1634aed78b5af6181e available at: https://gist.github.com/66ba63faaa75cc816135<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5665.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5665.dmg

**Comment by ctrlaltdavid** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: e2ee074b7a5890aa28e9ce1634aed78b5af6181e available at: https://gist.github.com/66ba63faaa75cc816135<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5665.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5665.dmg

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: e2ee074b7a5890aa28e9ce1634aed78b5af6181e available at: https://gist.github.com/66ba63faaa75cc816135<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5665.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5665.dmg

**Pull request merged**